test/integration/TestDocs/README.md

```diff
@@ -0,0 +1,6 @@
+# Steps to Compile & Execution of Integration Test suite 
```

Comment:
in previous PR , I mentioned if we can move this to docs folder with a sub-folder for `integration-test`.

Reply:
I am open to the document position but would suggest provide a link in top-level README.md

Reply:
if we are not moving to the original docs folder , then we should atleast rename "TestDocs" -> "docs" since we are already inside the `test` folder.